### PR TITLE
Introduce changelog and contributing files

### PR DESCRIPTION
**Pull request by svilenvelikov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+first endpoint and if it fails, switch to the next endpoint.
+* Retrying mechanism on the client - if the current endpoint fails to process a 
+client request with temporal unavailability, and there are no other functional 
+endpoints, the client library should support keeping the failed request for a 
+configurable period of time before returning the error to the caller; the 
+default time should be 0 seconds, which means fail immediately.
+* Allow configuring the HTTP connect and response timeout value so the user can 
+give up requests failing to return a result in the expected time.
+* Add X-Request-id to each client request
+* Consistent logging mechanism
```
Comment (mihailradkov) [inline CHANGELOG.md]: Maybe note it is structured logging in JSON format

Comment (svilenvelikov) [reply CHANGELOG.md]: Updated

**Comment by mihailradkov** [comment]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* Consume SPARQL Query results with Node streams.
+* Writing data (SPARQL/RDF4J protocols (POST+PUT)). 
+* Delete data (RDF4J protocols).
+* Context (i.e. named graph) & namespace management (RDF4J).
+* Transactions (RDF4J).
+* Compliant with [RDFJS](http://rdf.js.org/data-model-spec/) by plugable parsers.
+* Support multiple GDB endpoints to ensure high-availability - there should be a
+list of endpoints iterated by the client if one of them fails. Always test the 
+first endpoint and if it fails, switch to the next endpoint.
+* Retrying mechanism on the client - if the current endpoint fails to process a
```
Comment (desislava-hristova-ontotext) [inline CHANGELOG.md]: We dropped this requirement, so we should delete if from here.

**Pull request merged**